### PR TITLE
docs: Align existing splash screen info, minor docs updates

### PR DESCRIPTION
**Pull request by lukeblevins**
This work is the first component to a larger effort around aligning + improving our resources for setup/customization of splash screens:

### THIS PR
- Positioning existing content as a _manual_ option post-Resizetizer while still preserving docs relevant to that scenario

### Next steps
- Mentioning the toolkit's `ExtendedSplashScreen` as the recommended splash [solution for GTK](https://github.com/unoplatform/uno/issues/13338)
- Modifying bootstrapper repo page [Splash screen customization](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-splash-screen.html) to be a reference page for `AppManifest.js` properties: _Preventing it from being used first by everyday consumers of splash screen_
- Expanding what the toolkit repo has documented for `ExtendedSplashScreen`

_See: #14805_

### CODE REVIEW

File: doc/articles/features/working-with-assets.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Uno Platform automatically processes assets from your app's **Class Library Proj
 
 Support for automatic generation of assets multiple resolutions from SVG or PNG is also provided using [Uno.Resizetizer](xref:Uno.Resizetizer.GettingStarted).
 
-Platform specific assets such as [`BundleResource`](https://learn.microsoft.com/en-us/xamarin/ios/user-interface/controls/image) and [`AndroidAssets`](https://learn.microsoft.com/en-us/xamarin/android/app-fundamentals/resources-in-android/android-assets) are also supported on the heads, when required.
+Platform specific assets such as [`BundleResource`](https://learn.microsoft.com/xamarin/ios/user-interface/controls/image) and [`AndroidAssets`](https://learn.microsoft.com/xamarin/android/app-fundamentals/resources-in-android/android-assets) are also supported on the heads, when required.
```
Comment (agneszitte) [inline doc/articles/features/working-with-assets.md:11]:
```suggestion
Platform-specific assets such as [`BundleResource`](https://learn.microsoft.com/xamarin/ios/user-interface/controls/image) and [`AndroidAssets`](https://learn.microsoft.com/xamarin/android/app-fundamentals/resources-in-android/android-assets) are also supported on the heads, when required.
```

File: doc/articles/features/working-with-assets.md [hunk 2]
```diff
@@ -31,47 +31,54 @@ At the moment, the following image file types are supported as `Content` assets:
...
 <Image Source="ms-appx:///[MyApp]/Assets/MyImage.png" />
 ```
 
-You can also get assets directly using [StorageFile.GetFileFromApplicationUriAsync](file-management.md).
+You can also get assets directly using [StorageFile.GetFileFromApplicationUriAsync](xref:Uno.Features.FileManagement#support-for-storagefilegetfilefromapplicationuriasync).
 
 ## Qualify an asset
 
-On UWP, you can use qualifiers to load different assets depending on scale, language, etc.
+When developing Windows apps, developers can to load different assets at runtime based on attributes which qualify their visibility in the app UX. This allows you to tailor your app to different contexts to better suit users' hardware or language preferences.
```
Comment (agneszitte) [inline doc/articles/features/working-with-assets.md:55]:
```suggestion
When developing Windows apps, developers can load different assets at runtime based on attributes that qualify their visibility in the app UX. This allows you to tailor your app to different contexts to better suit users' hardware or language preferences.
```

File: doc/articles/splash-screen.md [hunk 3]
```diff
@@ -1,97 +1,68 @@
...
+* Refer to this [table](xref:Uno.Features.Assets#table-of-scales) to see values for the different scales required.
 
-    You can also just provide a single image named as `SplashScreen.png` without the `scale-000` qualifier.
+    * You can instead provide only a single image named `SplashScreen.png` without the `scale-000` qualifier.
 
     > [!NOTE]
     > Regardless if you provide a single image or multiple images, you would always refer to this image as `SplashScreen.png`.
 
-1. Add these images under the `Assets\` folder of the `MyApp` Class Library project, right-click on each image, go to `Properties` and set their build action as `Content`.
+* Add these images under the `Assets\` folder of the `MyApp` Class Library project, right-click on each image, go to `Properties` and set their build action as `Content`.
```
Comment (agneszitte) [inline doc/articles/splash-screen.md:40]:
```suggestion
* Add these images under the `Assets\` folder of the `MyApp` Class Library project, right-click on each image, go to `Properties`, and set their build action as `Content`.
```

File: doc/articles/splash-screen.md [hunk 4]
```diff
@@ -109,108 +80,115 @@ This article covers how to add a splash screen to your application if you don't
...
-    </ItemGroup>
-    ```
+* Make sure `splash.xml` is added as an `AndroidResource` in the Droid project file : `[Project-name].Droid.csproj`.
+
+    * This is not always done automatically, and may occur if `splash.xml` was created and added outside the IDE.
+        ```xml
+        <ItemGroup>
+            <AndroidResource Include="Resources\drawable\splash.xml" />
+        </ItemGroup>
+        ```
```
Comment (agneszitte) [inline doc/articles/splash-screen.md:93]:
```suggestion
* Make sure `splash.xml` is added as an `AndroidResource` in the Droid project file: `[Project-name].Droid.csproj`.

    * This is not always done automatically and may occur if `splash.xml` was created and added outside the IDE.
        ```xml
        <ItemGroup>
            <AndroidResource Include="Resources\drawable\splash.xml" />
        </ItemGroup>
        ```
```

File: doc/articles/splash-screen.md [hunk 5]
```diff
@@ -109,108 +80,115 @@ This article covers how to add a splash screen to your application if you don't
...
+        var UnoAppManifest = {
+            splashScreenImage: "Assets/SplashScreen.scale-200.png",
+            splashScreenColor: "transparent",
+            displayName: "SplashScreenSample"
+        }
+        ```
 
-## Table of scales
+    > [!NOTE]
+    > The section below contains properties that are optional. If nothing is assigned to them, the value of `splashScreenColor` will be used under both themes as the background color.
```
Comment (agneszitte) [inline doc/articles/splash-screen.md:173]:
```suggestion
    > The section below contains optional properties. If nothing is assigned to them, the value of `splashScreenColor` will be used under both themes as the background color.
```

File: doc/articles/splash-screen.md [hunk 6]
```diff
@@ -1,97 +1,68 @@
 ---
-uid: Uno.Development.Splashscreen
+uid: Uno.Development.SplashScreen
 ---
 
-# How to add a Splash Screen
+# How to manually add a splash screen
 
-This article covers how to add a splash screen to your application if you don't want to use Uno.Resizetizer to handle your Splash Screen.
+Projects created using Uno Platform 4.8 or later have the Uno.Resizetizer [package](https://www.nuget.org/packages/Uno.Resizetizer) installed by default. Simply provide an SVG file, and the tool handles the task of generating various image sizes. That package updates the build process to automate configuring a splash screen for each platform.
```
Comment (kazo0) [inline doc/articles/splash-screen.md:7]: Should we link to this part of the docs? https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=classlib%2CAndroid#unosplashscreen

Comment (kazo0) [reply doc/articles/splash-screen.md:7]: Nevermind I just saw you did that a bit later on :)

**Review by kazo0 — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

**Pull request merged**